File: serve/router/process.js
const express = require('express')
const router = express.Router()
const process_handler = require('../router_handler/process')

router.post('/list', process_handler.list)
router.post('/create', process_handler.create)
router.post('/update', process_handler.update)
router.post('/delete', process_handler.delete)
router.post('/batchDelete', process_handler.batchDelete)
router.post('/detail', process_handler.detail)
router.post('/updateTechnicianVideo', process_handler.updateTechnicianVideo)
router.post('/updateWebVideo', process_handler.updateWebVideo)

module.exports = router

File: python_serve/routers/process.py
from fastapi import APIRouter, Request
from handlers.process_handler import (
    list_processes, create_process, update_process, delete_process,
    batch_delete_processes, get_process_detail, update_technician_video, update_web_video
)

router = APIRouter(prefix="/api/process", tags=["process"])

@router.post("/list")
async def list_processes_route(request: Request):
    data = await request.json()
    return list_processes(data)

@router.post("/create")
async def create_process_route(request: Request):
    data = await request.json()
    return create_process(data)

@router.post("/update")
async def update_process_route(request: Request):
    data = await request.json()
    return update_process(data)

@router.post("/delete")
async def delete_process_route(request: Request):
    data = await request.json()
    return delete_process(data)

@router.post("/batchDelete")
async def batch_delete_processes_route(request: Request):
    data = await request.json()
    return batch_delete_processes(data)

@router.post("/detail")
async def get_process_detail_route(request: Request):
    data = await request.json()
    return get_process_detail(data)

@router.post("/updateTechnicianVideo")
async def update_technician_video_route(request: Request):
    data = await request.json()
    return update_technician_video(data)

@router.post("/updateWebVideo")
async def update_web_video_route(request: Request):
    data = await request.json()
    return update_web_video(data)
